### PR TITLE
Use monotonic clock for measuring observation time

### PR DESCRIPTION
**Pull request by jmgarnier**
See https://blog.dnsimple.com/2018/03/elapsed-time-with-ruby-the-right-way/

### CODE REVIEW

File: lib/scientist/observation.rb [hunk 1]
```diff
@@ -26,15 +26,15 @@ class Scientist::Observation
   def initialize(name, experiment, &block)
     @name       = name
     @experiment = experiment
-    @now        = Time.now
+    @now        = Process.clock_gettime(Process::CLOCK_MONOTONIC, :float_second)
```
[the remark this answers is not in the document]
Comment (jmgarnier) [reply lib/scientist/observation.rb]: 👍 Thanks :)

**Comment by zerowidth** [comment]
Nice, this makes a lot of sense. I'm fine with dropping support for <2.1 but wouldn't mind hearing from other maintainers.

**Comment by jmgarnier** [comment]
Thanks @zerowidth  ! Any chance other maintainers express their concerns, make up a decision?

**Comment by jbarnette** [comment]
> support for <2.1 

Yeah, this originally kept us from using the monotonic clock. @calavera was the last person I remember running 🔬 on 1.9, but that was years ago. I bet we're good here.

**Comment by zerowidth** [comment]
@jmgarnier thanks for the reminder. Go ahead and drop <2.1 support in this PR and I can release a new version.

**Comment by jmgarnier** [comment]
@zerowidth ✅ I haven't amended the `README`, let me know if I should add something.

**Comment by zerowidth** [comment]
Thanks @jmgarnier, I've released `v.1.20` with these changes.

**Pull request merged**